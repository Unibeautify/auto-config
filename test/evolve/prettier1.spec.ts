import { testGenetic } from "../testGenetic";
import prettier from "@unibeautify/beautifier-prettier";

testGenetic("prettier1", {
  beautifiers: [prettier],
  language: "JavaScript",
  originalText: `var foo = {
    bar: "baz",
    qux: "quux",
};

var bar = {
    bar: "baz",
    qux: "quux"
};
`,
  desiredText: `/** @format */

var foo = {
  bar: "baz",
  qux: "quux",
}

var bar = {
  bar: "baz",
  qux: "quux",
}
`,
});
