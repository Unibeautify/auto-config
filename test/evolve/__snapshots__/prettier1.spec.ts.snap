// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prettier1: beautifiedText 1`] = `
/** @format */

var foo = {
  bar: "baz",
  qux: "quux",
}

var bar = {
  bar: "baz",
  qux: "quux",
}

`;

exports[`prettier1: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "Prettier",
    ],
    "end_with_comma": true,
    "end_with_semicolon": false,
    "pragma_insert": true,
  },
}
`;

exports[`prettier1: fitness 1`] = `2`;
